datafusion/sqllogictest/test_files/create_external_table.slt

```diff
@@ -228,3 +228,13 @@ OPTIONS (
         format.delimiter '|',
         has_header false,
         compression gzip);
+
+# Create an external parquet table and infer schema to order by
+
+# query should succeed
+statement ok
+CREATE EXTERNAL TABLE t STORED AS parquet LOCATION '../../parquet-testing/data/alltypes_plain.parquet' WITH ORDER (id);
```

Comment:
Can you also add a test that shows the table is actually ordered correctly?

Reply:
Can do